.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on: push
+
+jobs:
+  test:
+    name: Go ${{ matrix.go-version }} on Ubuntu
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.12.x, 1.13.x]
+        platform: [ubuntu-latest]
+
+    env:
+      KAFKA_PEERS: localhost:9091,localhost:9092,localhost:9093,localhost:9094,localhost:9095
+      TOXIPROXY_ADDR: http://localhost:8474
+      KAFKA_INSTALL_ROOT: /home/runner/kafka
+      KAFKA_HOSTNAME: localhost
+      DEBUG: true
+      KAFKA_VERSION: 2.3.0
```

Comment:
🤔 where is going to be defined the different kafka server versions?

Reply:
Updated this PR to include kafka-version in build matrix.